private/TupleManipulation.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+namespace TupleManipulation {
```

Comment:
I think I'll rename this whole file my own way (I took the snippets from somewhere else).